### ISSUE TITLE
docker拉取镜像失败，只能拉取部分，导致服务端运行失败

### ISSUE THREAD

**Issue by Xuyaoyan** [modelscope/FunASR#2037]
Notice: In order to resolve issues more efficiently, please raise issue following the template.
（注意：为了更加高效率解决您遇到的问题，请按照模板提问，补充细节）

## 🐛 Bug

镜像拉取失败，只能拉取部分内容

funasr-runtime-sdk-gpu-0.1.1: Pulling from funasr_repo/funasr
d5fd17ec1767: Pulling fs layer
aa1c1095ee01: Pulling fs layer
4b5a4c7a4fbf: Pulling fs layer
13352ad804a8: Pulling fs layer
d3dba242f337: Pulling fs layer
b8e20393c8a7: Pulling fs layer
f36d862e19f9: Pulling fs layer
527a82de0a82: Pull complete
bcf7babe7ac9: Pull complete
e4cee52b04df: Pull complete
464033584828: Pull complete
86ea9d50baee: Pull complete
8fb0032e821c: Pull complete
7513c55cd0f8: Pull complete
db5b341d7ade: Pull complete
86585f484edd: Pull complete
3c2ad08d695c: Pull complete
82eb8d68d70a: Pull complete
2b198bb4ab11: Pull complete
922d5e5377c8: Pull complete
943cab03b991: Pull complete
8b01b23b03af: Pull complete
fc1dba29c373: Pull complete
fead719ec0a5: Pull complete
b8b3b2066984: Pull complete
39fe24ba71ba: Pull complete
0b8c2aec6529: Pull complete
e4b6d053a883: Pull complete
9b4c8bb5936f: Pull complete
e5a2ea2a1088: Pull complete
a613ad16291d: Pull complete
64d5795557dc: Pull complete
b15a600ea35e: Pull complete
8d8e5db54a83: Pull complete
Digest: sha256:6c9455bb9eebf0d813d3caab20eac7ca79902f53256af58267c7f26bc2087742
Status: Downloaded newer image for registry.cn-hangzhou.aliyuncs.com/funasr_repo/funasr:funasr-runtime-sdk-gpu-0.1.1
registry.cn-hangzhou.aliyuncs.com/funasr_repo/funasr:funasr-runtime-sdk-gpu-0.1.1

最开头几个知道拉取结束一直处于Pull fs layer的状态

导致运行fsunasr-wss-server之后，无法成功运行
部分log

[W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema:
aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
%input.947 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.945, %5532, %5533, %5534), scope: __module.model.encoders/__module.model.encoders.46/__module.model.encoders.46.self_attn/__module.model.encoders.46.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0

[W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema:
aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
%input.967 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.965, %5646, %5647, %5648), scope: __module.model.encoders/__module.model.encoders.47/__module.model.encoders.47.self_attn/__module.model.encoders.47.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0

[W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema:
aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
%input.987 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.985, %5760, %5761, %5762), scope: __module.model.encoders/__module.model.encoders.48/__module.model.encoders.48.self_attn/__module.model.encoders.48.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0

Killed
I20240825 14:45:06.462719   139 funasr-wss-server.cpp:297] Failed to download model from modelscope. If you set local asr model path, you can ignore the errors.
E20240825 14:45:06.471107   139 funasr-wss-server.cpp:301] /workspace/models/damo/speech_paraformer-large-vad-punc_asr_nat-zh-cn-16k-common-vocab8404-pytorch/model_blade.torchscript do not exists.

### To Reproduce

Steps to reproduce the behavior (**always include the command you ran**):

1. Run cmd 
sudo docker pull \
  registry.cn-hangzhou.aliyuncs.com/funasr_repo/funasr:funasr-runtime-sdk-gpu-0.1.1
mkdir -p ./funasr-runtime-resources/models
sudo docker run --gpus=all -p 10098:10095 -it --privileged=true \
  -v $PWD/funasr-runtime-resources/models:/workspace/models \
  registry.cn-hangzhou.aliyuncs.com/funasr_repo/funasr:funasr-runtime-sdk-gpu-0.1.1

cd FunASR/runtime
nohup bash run_server.sh \
  --download-model-dir /workspace/models \
  --vad-dir damo/speech_fsmn_vad_zh-cn-16k-common-onnx \
  --model-dir damo/speech_paraformer-large-vad-punc_asr_nat-zh-cn-16k-common-vocab8404-pytorch  \
  --punc-dir damo/punc_ct-transformer_cn-en-common-vocab471067-large-onnx \
  --lm-dir damo/speech_ngram_lm_zh-cn-ai-wesp-fst \
  --itn-dir thuduj12/fst_itn_zh \
  --hotword /workspace/models/hotwords.txt > log.txt 2>&1 &

2. See error

![image](https://github.com/user-attachments/assets/e7cdc76d-dca6-44b8-9e75-1c69fbd43200)

![image](https://github.com/user-attachments/assets/dab1cf51-4afe-4ffa-b0d3-384da7a9cdfc)

 - OS (e.g., Linux): Ubuntu20.04
 - FunASR Version (e.g., 1.0.0): funasr-runtime-sdk-gpu-0.1.1
 - ModelScope Version (e.g., 1.11.0):
 - PyTorch Version (e.g., 2.0.0):
 - How you installed funasr (`pip`, source):
 - Python version:
 - GPU (e.g., V100M32):GeForce 3060 Ti
 - CUDA/cuDNN version (e.g., cuda11.7):12.4
 - Docker version (e.g., funasr-runtime-sdk-cpu-0.4.1):funasr-runtime-sdk-gpt-0.1.1

**Comment by lyblsgo** [comment]
无法复现你的问题，请check网络或者关注给docker分配的资源

**Comment by lyblsgo** [comment]
Let's close this issue for now. If needed, reopen it.
